concepts/05 Widgets/Toast/10 Customize the Content.md

```diff
@@ -155,6 +253,122 @@ If you need to render different templates depending on a specific condition, def
          // ...
      })
 
+##### Vue
+
+    <template>
+        <div>
+            <DxToast
+                type="custom"
+                :visible.sync="isVisible"
+                :contentTemplate="contentTemplate">
+                <template #green>
+                    <p style="background-color:green">Green template</p>
+                </template>
+                <template #blue>
+                    <p style="background-color:blue">Blue template</p>
+                </template>
+            </DxToast>
+            <DxButton
+                text="Change the Toast Template"
+                @click="onClick"
+            />
+        </div>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxToast } from 'devextreme-vue/toast';
+    import { DxButton } from 'devextreme-vue/button';
+
+    export default {
+        components: {
+            DxToast,
+            DxButton
+        },
+        data() {
+            return {
+                isVisible: false,
+                contentTemplate: "green"
+            };
+        },
+        methods: {
+            onClick() {
+                this.contentTemplate = this.contentTemplate === "green" ? "blue" : "green";
+                this.isVisible = true;
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Toast } from 'devextreme-react/toast';
+    import { Button } from 'devextreme-react/button';
+
+    const greenTemplate = () => {
+        return (
+            <p style={{backgroundColor: 'green'}}>Green template</p>
+        );
+    }
+
+    const blueTemplate = () => {
+        return (
+            <p style={{backgroundColor: 'blue'}}>Blue template</p>
+        );
+    }
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+
+            this.state = {
+                isVisible: false,
+                renderContent: greenTemplate
+            };
+
+            this.onClick = this.onClick.bind(this);
+            this.onHiding = this.onHiding.bind(this);
+        }
+
+        onClick() {
+            this.setState({
+                isVisible: true,
+                renderContent: this.state.renderContent === greenTemplate ? blueTemplate : greenTemplate
```

Comment:
Looks interesting :) Does it actually work?

Reply:
yeap, I test all non-trivial snippets